Index: app/src/main/java/com/example/giftly/GiftSignup.java
===================================================================
diff --git a/app/src/main/java/com/example/giftly/GiftSignup.java b/app/src/main/java/com/example/giftly/GiftSignup.java
new file mode 100644
--- /dev/null	
+++ b/app/src/main/java/com/example/giftly/GiftSignup.java	
@@ -0,0 +1,14 @@
+package com.example.giftly;
+
+import androidx.appcompat.app.AppCompatActivity;
+
+import android.os.Bundle;
+
+public class GiftSignup extends AppCompatActivity {
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_gift_signup);
+    }
+}
\ No newline at end of file
