Index: app/src/main/java/com/example/giftly/LoginScreen.java
===================================================================
diff --git a/app/src/main/java/com/example/giftly/LoginScreen.java b/app/src/main/java/com/example/giftly/LoginScreen.java
deleted file mode 100644
--- a/app/src/main/java/com/example/giftly/LoginScreen.java	
+++ /dev/null	
@@ -1,67 +0,0 @@
-package com.example.giftly;
-
-import androidx.annotation.NonNull;
-import androidx.appcompat.app.ActionBar;
-import androidx.appcompat.app.AppCompatActivity;
-
-import android.content.Intent;
-import android.os.Bundle;
-import android.view.MenuItem;
-import android.view.View;
-import android.widget.Button;
-import android.widget.EditText;
-import android.widget.Toast;
-
-public class LoginScreen extends AppCompatActivity {
-
-    //Variables
-    EditText username, password;
-    Button btnlogin;
-    DB_Helper DB;
-
-    @Override
-    protected void onCreate(Bundle savedInstanceState) {
-        super.onCreate(savedInstanceState);
-        setContentView(R.layout.activity_login_screen);
-
-        username = (EditText) findViewById(R.id.usernm1);
-        password = (EditText) findViewById(R.id.userpasswrd1);
-        btnlogin = (Button) findViewById(R.id.signsbutton1);
-        DB = new DB_Helper(this);
-
-        //Back button
-        ActionBar actionBar = getSupportActionBar();
-        actionBar.setDisplayHomeAsUpEnabled(true);
-
-        btnlogin.setOnClickListener(new View.OnClickListener() {
-            @Override
-            public void onClick(View view) {
-                String user = username.getText().toString();
-                String pass = password.getText().toString();
-
-                if(user.equals("")||pass.equals(""))
-                    Toast.makeText(LoginScreen.this,"Please enter all the fields", Toast.LENGTH_SHORT).show();
-                else{
-                    Boolean checkuserpass = DB.checkusernamepassword(user, pass);
-                    if(checkuserpass == true){
-                        Toast.makeText(LoginScreen.this,"Sign in Successfull", Toast.LENGTH_SHORT).show();
-                        Intent intent = new Intent(getApplicationContext(), HomeScreen.class);
-                        startActivity(intent);
-                    }else{
-                        Toast.makeText(LoginScreen.this,"Invalid Credentials", Toast.LENGTH_SHORT).show();
-                    }
-                }
-            }
-        });
-    }
-
-    @Override
-    public boolean onOptionsItemSelected(@NonNull MenuItem item) {
-        switch (item.getItemId()) {
-            case android.R.id.home:
-                this.finish();
-                return true;
-        }
-        return super.onOptionsItemSelected(item);
-    }
-}
\ No newline at end of file
Index: app/src/main/java/com/example/giftly/DB_Helper.java
===================================================================
diff --git a/app/src/main/java/com/example/giftly/DB_Helper.java b/app/src/main/java/com/example/giftly/DB_Helper.java
deleted file mode 100644
--- a/app/src/main/java/com/example/giftly/DB_Helper.java	
+++ /dev/null	
@@ -1,58 +0,0 @@
-package com.example.giftly;
-
-import android.content.ContentValues;
-import android.content.Context;
-import android.database.Cursor;
-import android.database.sqlite.SQLiteDatabase;
-import android.database.sqlite.SQLiteOpenHelper;
-
-public class DB_Helper extends SQLiteOpenHelper {
-
-    public static final String DBNAME = "Login.db";
-
-    public DB_Helper(Context context) {
-        super(context, "Login.db", null, 1);
-    }
-
-    @Override
-    public void onCreate(SQLiteDatabase MyDB) {
-        MyDB.execSQL("create Table users(username TEXT primary key, password TEXT)");
-    }
-
-    @Override
-    public void onUpgrade(SQLiteDatabase MyDB, int i, int i1) {
-        MyDB.execSQL("drop Table if exists users");
-    }
-
-    public Boolean insertData(String username, String password){
-        SQLiteDatabase MyDB = this.getWritableDatabase();
-        ContentValues contentValues = new ContentValues();
-        contentValues.put("username", username);
-        contentValues.put("password", password);
-        long result = MyDB.insert("users", null, contentValues);
-        if(result == -1){
-            return false;
-        }
-        else{
-            return true;
-        }
-    }
-
-    public Boolean checkusername(String username){
-        SQLiteDatabase MyDB = this.getWritableDatabase();
-        Cursor cursor = MyDB.rawQuery("Select * from users where username = ?", new String[] {username});
-        if(cursor.getCount() > 0)
-            return true;
-        else
-            return false;
-    }
-
-    public Boolean checkusernamepassword(String username, String password){
-        SQLiteDatabase MyDB = this.getWritableDatabase();
-        Cursor cursor = MyDB.rawQuery("Select * from users where username = ? and password = ?", new String[] {username, password});
-        if(cursor.getCount() > 0)
-            return true;
-        else
-            return false;
-    }
-}
Index: .idea/gradle.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+><?xml version=\"1.0\" encoding=\"UTF-8\"?>\r\n<project version=\"4\">\r\n  <component name=\"GradleMigrationSettings\" migrationVersion=\"1\" />\r\n  <component name=\"GradleSettings\">\r\n    <option name=\"linkedExternalProjectsSettings\">\r\n      <GradleProjectSettings>\r\n        <option name=\"testRunner\" value=\"GRADLE\" />\r\n        <option name=\"distributionType\" value=\"DEFAULT_WRAPPED\" />\r\n        <option name=\"externalProjectPath\" value=\"$PROJECT_DIR$\" />\r\n        <option name=\"gradleJvm\" value=\"JDK\" />\r\n        <option name=\"modules\">\r\n          <set>\r\n            <option value=\"$PROJECT_DIR$\" />\r\n            <option value=\"$PROJECT_DIR$/app\" />\r\n          </set>\r\n        </option>\r\n      </GradleProjectSettings>\r\n    </option>\r\n  </component>\r\n</project>
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/gradle.xml b/.idea/gradle.xml
--- a/.idea/gradle.xml	
+++ b/.idea/gradle.xml	
@@ -7,7 +7,7 @@
         <option name="testRunner" value="GRADLE" />
         <option name="distributionType" value="DEFAULT_WRAPPED" />
         <option name="externalProjectPath" value="$PROJECT_DIR$" />
-        <option name="gradleJvm" value="JDK" />
+        <option name="gradleJvm" value="11" />
         <option name="modules">
           <set>
             <option value="$PROJECT_DIR$" />
